### PR TITLE
Fix falsy error message in bash script

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by warunicorn19 — APPROVED** [review]

**Review by warunicorn19 — APPROVED** [review]

**Comment by leecalcote** [comment]
Please label this PR

**Comment by leecalcote** [comment]
In removing the instructions for users in the error message, are we missing an opportunity to guide them to resolution of the issue?

File: meshery [hunk 1]
```diff
@@ -117,7 +117,7 @@ curl -L ${URL} -o ${PWD}/meshery.zip
 
 validFile=`file ${PWD}/meshery.zip | grep 'Zip archive data'`
 if [ -z "$validFile" ] ; then
-  printf "Unable to get latest meshery install package. Set MESHERY_VERSION env var and re-run. For example: export MESHERY_VERSION=v0.1.6\n"
+  printf "Unable to download meshery.zip file from ${URL}"
```
Comment (hexxdump) [inline meshery:120]:
```suggestion
  printf "Unable to download meshery.zip file from ${URL}. Set MESHERY_VERSION environment variable to a different version and re-run. For example: export MESHERY_VERSION=v0.1.6\n"
```

**Review by hexxdump — COMMENTED** [review]
we should retain the error message about environment variable, in case meshery.zip file does not exist for a particular version.

File: meshery (hunk 1 above)
Comment (leecalcote) [reply meshery:120]: Roger that. Thoughts, @Revolyssup?

Comment (Revolyssup) [reply meshery:120]: LGTM

Comment (leecalcote) [reply meshery:120]: When ready, commit suggestion, if you would.

**Review by hexxdump — APPROVED** [review]

**Pull request merged**